geoips/interfaces/base.py

```diff
@@ -662,14 +666,14 @@ def get_plugin(self, name):
                 f" attempting to access the correct plugin name)"
             ) from resp
         # Convert the module into an object
-        return plugin_module_to_obj(name, module)
+        return plugin_module_to_obj(self, name, module)
```

Comment:
If we move plugin_module_to_obj onto this class as a classmethod, this line can be reverted.

Reply:
See https://github.com/NRLMMD-GEOIPS/geoips/pull/351#discussion_r1355580586

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/369